ellie/monopoly.py

```diff
@@ -0,0 +1,154 @@
+from random import randint
+
+class Player:
+    def __init__(self, player_name, player_token):
+        self.name = player_name
+        self.token = player_token
+        self.money = 1500
+        self.properties = []
+        self.position = 0
+        self.in_jail = False
+        self.is_turn = False
+    
+    def __repr__(self):
+        return f"This player, {self.name}, is playing as the {self.token}. \n{self.name} has ${self.money}, is currently on {board[self.position]}, and owns these properties: {self.properties}"
+
+    def roll_dice(self):
+        # Roll two dice and return the results
+        die1 = randint(1,6)
+        die2 = randint(1,6)
+        result = die1 + die2
+        print(f"{self.name} rolled a {result} with a {die1} and a {die2}!")
+        return [die1, die2, result]
+    
+    def update_position(self):
+        # Check the dice roll and update position on the board.
+        result = self.roll_dice()
+        while result[2] > 0:
+            self.position += 1
+            result[2] -= 1
+            # Wrap around board
```

Comment:
A future version might consider a `Board` class to handle movement, wrapping around, containing properties, etc. That would let the `Player` class stay concerned with tracking the state of the player only and not have to also know anything about the board.

(e.g. you could make an alternative Monopoly that has 60 spaces without having to modify the `Player` class since it would be compatible with any type of board that implements the right methods)

Reply:
Oooh, good idea, I'll give that a shot.